test/integration/user_can_create_account_test.rb

```diff
@@ -17,7 +17,7 @@ class UserCanCreateAccountTest < ActionDispatch::IntegrationTest
     fill_in "Email address", with: "brenna@awesome.com"
     fill_in "Street address", with: "123 Maple Drive"
     fill_in "City", with: "Denver"
-    fill_in "State", with: "CO"
+    select('Colorado', :from => 'user_state')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.